common/live/prd/infra/dev-natarajaya/terraform.tfvars

```diff
@@ -0,0 +1,19 @@
+# ↓ Module metadata
```

Comment:
I think this should be in `common/live/dev/natarajaya`.

Reply:
Nevermind, I replied with more details in https://github.com/gpii-ops/gpii-infra/pull/60#discussion_r205665333